public/index.ejs

```diff
@@ -36,7 +36,7 @@
     <title><%= isDevelopment ? "HeLx UI" : "%META_TITLE%" %></title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
I don't understand why you changed this? The website is built on React (which is Javascript), and the content of noscript displays when Javascript is disabled. So it's more or less just an informational error message for the user that the website is unusable unless they enable JavaScript.

Feel free to improve the error message but changing this to HeLx UI would be confusing.

Reply:
This text was showing up on Google Search. We can change it.